core/base/ftmTree/FTMTree_MT.cpp

```diff
@@ -1006,18 +1000,7 @@ SimplexId FTMTree_MT::trunkCTSegmentation(const vector<SimplexId> &trunkVerts,
 #ifdef TTK_ENABLE_OPENMP
 #pragma omp taskwait
```

Comment:
do we need to leave that here? @CharlesGueunet

Reply:
a quick test without those lead me to a segfault, so I assume they are needed ^^

We need to ensure all tasks are finished before leaving this context, and I assume the `return` instruction is not enough in a `task` block to ensure so.